lazyflow/utility/io_util/OMEZarrStore.py

```diff
@@ -0,0 +1,181 @@
+###############################################################################
+#   ilastik: interactive learning and segmentation toolkit
+#
+#       Copyright (C) 2011-2024, the ilastik developers
+#                                <team@ilastik.org>
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# In addition, as a special exception, the copyright holders of
+# ilastik give you permission to combine ilastik with applets,
+# workflows and plugins which are not covered under the GNU
+# General Public License.
+#
+# See the LICENSE file for details. License information is also available
+# on the ilastik web site at:
+# 		   http://ilastik.org/license.html
+###############################################################################
+import json
+import logging
+from typing import Optional, Dict
+
+import jsonschema
+import vigra
+from zarr.core import Array as ZarrArray
+from zarr.storage import FSStore, LRUStoreCache
+
+from lazyflow import rtype
+from lazyflow.utility import Timer
+from lazyflow.utility.io_util.multiscaleStore import MultiscaleStore, Multiscale, DEFAULT_LOWEST_SCALE_KEY
+
+logger = logging.getLogger(__name__)
+
+OME_ZARR_V_0_4_ARGS = {
+    "dimension_separator": "/",
```

Comment:
This is just a preference, but I personally like to use `dict(k1=v1, k2=v2, ...)` instead of dict literals, but only when all keys are function kwargs: this makes it easier to change code in case you need to inline these kwargs later (no need to remove quotes and replace `:` with `=`).

Of course, feel free to ignore this, it's not really important.